rpcserverhelp.go

```diff
@@ -488,6 +488,17 @@ var helpDescsEnUS = map[string]string{
 	"setgenerate-generate":     "Use true to enable generation, false to disable it",
 	"setgenerate-genproclimit": "The number of processors (cores) to limit generation to or -1 for default",
 
+	// AddMiningAddrCmd help.
```

Comment:
Please alphabetize these too so it's all in the same order it's defined in the other file(s).